kubernetes-templates/weave-cloud/0/rancher-compose.yml

```diff
@@ -0,0 +1,10 @@
+.catalog:
+  name: "Weave Cloud Suite"
+  description: "All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"
```

Comment:
Maybe change:

"All you need to implement Continuous Delivery with hosted Prometheus monitoring and visual distributed system analysis/debug tools"

"Weave Cloud is a hosted add-on to Kubernetes which provides Continuous Delivery, Prometheus for Monitoring and a visual dashboard for troubleshooting & debugging microservices. This package contains the agents which connect your cluster to Weave Cloud. You will need a service token which you can get from https://cloud.weave.works/"